FitVid/src/index.js

```diff
@@ -11,29 +11,32 @@ import { HistoryProvider } from "./Context/history-context";
 import { WatchLaterProvider } from "./Context/watch-later-context";
 import { PlayListModalProvider } from "./Context/playlist-modal-context";
 import { PlayListsProvider } from "./Context/playlist-context";
+import { CategoryProvider } from "./Context/category-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <PlayListsProvider>
-      <PlayListModalProvider>
-        <WatchLaterProvider>
-          <HistoryProvider>
-            <LikedProvider>
-              <UserProvider>
-                <ToastProvider>
-                  <Router>
-                    <App />
-                  </Router>
-                </ToastProvider>
-              </UserProvider>
-            </LikedProvider>
-          </HistoryProvider>
-        </WatchLaterProvider>
-      </PlayListModalProvider>
-    </PlayListsProvider>
+    <Router>
+      <CategoryProvider>
+        <PlayListsProvider>
+          <PlayListModalProvider>
+            <WatchLaterProvider>
+              <HistoryProvider>
+                <LikedProvider>
+                  <UserProvider>
+                    <ToastProvider>
+                      <App />
+                    </ToastProvider>
+                  </UserProvider>
+                </LikedProvider>
+              </HistoryProvider>
+            </WatchLaterProvider>
+          </PlayListModalProvider>
+        </PlayListsProvider>
+      </CategoryProvider>
+    </Router>
```

Comment:
This looks so cool 🚀.

Reply:
Cool kaise?😂

Reply:
Means it is very clear what each line is doing and also looks like a arrow.😂